src/components/Loans/ClosedLoans/ClosedLoans.js

```diff
@@ -324,7 +337,7 @@ class ClosedLoans extends React.Component {
               >
                 {anonymizeString}
               </Button>
-              <ExportCsv data={loans} excludeKeys={['id', 'userId', 'itemId']} />
+              <ExportCsv data={recordsToCSV} onlyFields={this.columnHeadersMap} />
```

Comment:
Could you probably place props on separate lines?

Reply:
@Yurii-Danylenko Done